### PR TITLE
Overall review of Core docs 3.0.0 - Closes #677

### LINKED ISSUE #677: Overall review of Core docs 3.0.0
Overall review of entire documentation to include the structure, tone of voice, style, grammar, etc, etc

Parent issue #665

### PR DESCRIPTION
**Pull request by chris529**
Overall review of structure, tone of voice, style, grammar, etc, etc

Closes #677

### CODE REVIEW

File: modules/ROOT/pages/interact-with-the-api.adoc [hunk 1]
```diff
@@ -7,6 +7,7 @@ Mona Bärenfänger <mona@lightcurve.io>
 :page-previous: /lisk-core/index.html
 :page-previous-title: Introduction
 
+:url_api: https://swagger.io/
```
Comment (Tschakki) [inline modules/ROOT/pages/interact-with-the-api.adoc]: Why are you changing the link here?

File: modules/ROOT/pages/interact-with-the-api.adoc [hunk 2]
```diff
@@ -44,11 +44,11 @@ Depending on their preferences, node operators may wish to use <<javascript,Lisk
...
-. *LSK Token Holders* mostly interact with the network through Graphical User Interfaces such as wallet applications such as {url_lisk_wallet}[Lisk Desktop or Lisk Mobile^].
+. *LSK Token Holders* mostly interact with the network through Graphical User Interfaces (GUIs),  such as wallet applications such as {url_lisk_wallet}[Lisk Desktop or Lisk Mobile^].
 
 == How to interact with the network?
 
-The diagram shown below illustrates the different possibilities of interacting with the network. It is possible to use the regular interaction method through the lisk-api-client library or deploy different methods such as the Lisk Commander CLI, Lisk Hub Wallet, or the Lisk Explorer. Lastly, it is possible to write your own API request in any programming language.
+The diagram shown below illustrates the different possibilities of interacting with the network. It is possible to use the regular interaction method through the Lisk-api-client library or deploy different methods such as the Lisk Commander CLI, Lisk Hub Wallet, or the Lisk Explorer. Lastly, it is possible to write your own API request in any programming language.
```
Comment (Tschakki) [inline modules/ROOT/pages/interact-with-the-api.adoc]:
```suggestion
The diagram shown below illustrates the different possibilities of interacting with the network. It is possible to use the regular interaction method through the `lisk-api-client` library or deploy different methods such as the Lisk Commander CLI, Lisk Hub Wallet, or the Lisk Explorer. Lastly, it is possible to write your own API request in any programming language.
```

File: modules/ROOT/pages/management/application.adoc [hunk 3]
```diff
@@ -131,7 +131,7 @@ Argument `network` is required and can be `devnet`, `betanet` or any other netwo
 === Update config
 
 This next script is designed to keep track of all changes introduced in Lisk over time in different versions.
-If you have one config file existing in any specific version, and you wish to make it compatible with other versions of the Lisk, then execute this update config script shown below:
+If you have one config file existing in any specific version, and you wish to make it compatible with other versions of Lisk, then execute this update config script shown below:
```
Comment (Tschakki) [inline modules/ROOT/pages/management/application.adoc]:
```suggestion
If you have one config file existing in any specific version, and you wish to make it compatible with other versions of Lisk Core, then execute this update config script shown below:
```

File: modules/ROOT/pages/management/logs.adoc [hunk 4]
```diff
@@ -12,7 +12,7 @@ For monitoring or debugging your node, Lisk Core tracks all activities that occu
 These log messages are grouped in different log levels.
 Hence, this makes it easy to define the level of the log details.
 
-{url_bunyan}[Bunyan^] is used for the logging library, as this allows simple and fast JSON logging for Node.js services.
+{url_bunyan}[Bunyan^] is used for the logging library, as this allows simple and fast JSON logging for Node.JS services.
```
Comment (Tschakki) [inline modules/ROOT/pages/management/logs.adoc]:
```suggestion
{url_bunyan}[Bunyan^] is used for the logging library, as this allows simple and fast JSON logging for Node.js services.
```

It shoud always have `js` lowercase.

File: modules/ROOT/pages/management/source.adoc [hunk 5]
```diff
@@ -106,7 +106,7 @@ Argument `network` is required, and it may be either `devnet`, `betanet` or any
 === Update config
 
 This next script is designed to keep track of all changes introduced in Lisk over time in different versions.
-If you have one config file existing in any specific version, and you wish to make it compatible with other versions of the Lisk, then execute the update config script shown below:
+If you have one config file existing in any specific version, and you wish to make it compatible with other versions of Lisk, then execute the update config script shown below:
```
Comment (Tschakki) [inline modules/ROOT/pages/management/source.adoc]:
```suggestion
If you have one config file existing in any specific version, and you wish to make it compatible with other versions of Lisk Core, then execute the update config script shown below:
```

File: modules/ROOT/pages/setup/application.adoc [hunk 6]
```diff
@@ -113,17 +113,17 @@ If this is not possible, please refer to the {url_dev_forum}[Developer's Forum^]
...
 ----
 cd lisk-beta
 bash lisk.sh status
 ----
 
-For further information and how to administer your Lisk node, please see the xref:{url_admin_binary}[Management Section].
+For further information and how to administer your Lisk node, please see the xref:{url_admin_binary}[Management section].
 
-If you are not running Lisk locally, you will need to follow the xref:{url_config_api_access}[Configuration - API] document to enable access.
+If you are not running Lisk locally, you will need to follow the xref:{url_config_api_access}[API - Configuration] document to enable access.
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/application.adoc]:
```suggestion
If you are not running Lisk locally, you will need to follow the xref:{url_config_api_access}[Control API access] guide to enable access.
```

File: modules/ROOT/pages/setup/application.adoc [hunk 7]
```diff
@@ -113,17 +113,17 @@ If this is not possible, please refer to the {url_dev_forum}[Developer's Forum^]
...
+It is possible to verify that your Lisk node is up and running, by executing the following command:
 
 [source,bash]
 ----
 cd lisk-beta
 bash lisk.sh status
 ----
 
-For further information and how to administer your Lisk node, please see the xref:{url_admin_binary}[Management Section].
+For further information and how to administer your Lisk node, please see the xref:{url_admin_binary}[Management section].
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/application.adoc]:
```suggestion
For further information and how to administer your Lisk node, please see the xref:{url_admin_binary}[Application commands] page.
```

File: modules/ROOT/pages/setup/commander.adoc [hunk 8]
```diff
@@ -129,7 +129,7 @@ If you choose this option, the setup process will take significantly longer.
 
 == Post-install
 
-After the installation is completed, check which ports Lisk Core is listening to by checking the status using the following commands below:
+After the installation is completed, check which ports Lisk Core is listening to by checking the status using the applicable command shown below:
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/commander.adoc]:
```suggestion
After the installation is completed, check which ports Lisk Core is listening to.
To do that, check the status of Lisk Core by using the command shown below:
```

File: modules/ROOT/pages/setup/commander.adoc [hunk 9]
```diff
@@ -151,11 +151,11 @@ lisk core:status lisk-testnet
 --
 ====
 
-Check you network settings to verify the corresponding ports are open.
+Check your network settings to verify the corresponding ports are open.
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/commander.adoc]:
```suggestion
Then check the network settings to verify the corresponding ports are open.
```

File: modules/ROOT/pages/setup/commander.adoc [hunk 10]
```diff
@@ -151,11 +151,11 @@ lisk core:status lisk-testnet
...
 
 In addition it is also recommended to set up a xref:{url_config_logrotation}[log rotation].
 
-If you are not running Lisk locally, you will need to follow the xref:{url_config_api_access}[Configuration - API] guide to enable access.
+If you are not running Lisk locally, you will need to follow the xref:{url_config_api_access}[API - Configuration] guide to enable access.
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/commander.adoc]:
```suggestion
If you are not running Lisk Core locally, you will need to follow the xref:{url_config_api_access}[Control API access] guide to enable access.
```

File: modules/ROOT/pages/setup/docker.adoc [hunk 11]
```diff
@@ -56,14 +56,14 @@ sudo apt-get install curl make
 MacOS::
 +
 --
-Please refer to {url_docker_install_mac}[Docker installation for Mac^].
+Please refer to {url_docker_install_mac}[docker installation for Mac^].
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/docker.adoc]:
```suggestion
Please refer to {url_docker_install_mac}[Docker installation for Mac^].
```

Docker is always capitalized.

File: modules/ROOT/pages/setup/docker.adoc [hunk 12]
```diff
@@ -56,14 +56,14 @@ sudo apt-get install curl make
...
 Please note that Docker for Mac already includes Docker Compose.
 Install `make` using {url_xcode}[XCode^].
 --
 Windows::
 +
 --
-Please refer to {url_docker_install_windows}[Docker installation for Windows^].
+Please refer to {url_docker_install_windows}[docker installation for Windows^].
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/docker.adoc]:
```suggestion
Please refer to {url_docker_install_windows}[Docker installation for Windows^].
```

File: modules/ROOT/pages/setup/docker.adoc [hunk 13]
```diff
@@ -123,7 +123,7 @@ cd lisk-core/docker                                <3>
 git checkout v3.0.0-beta.0                         <4>
 ----
 
-<1> Switch to lisk user.
+<1> Switch to Lisk user.
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/docker.adoc]:
```suggestion
<1> Switch to the `lisk` user.
```

File: modules/ROOT/pages/setup/index.adoc [hunk 14]
```diff
@@ -19,7 +19,7 @@ There are four available distributions of Lisk Core which are described below. S
 === Application
 
 This is the *default* procedure to setup Lisk Core.
-The xref:{url_binary_installation}[Application installation] offers an easy and automated approach to set up Lisk Core, which includes almost completely automated update scripts, together with a selection of tools to assist with seamlessly maintaining a Lisk Node. This is the recommended installation for the less experienced user.
+The xref:{url_binary_installation}[Application installation] offers an easy and automated approach to set up Lisk Core. This includes almost completely automated update scripts, together with a selection of tools to assist with seamlessly maintaining a Lisk node. This is the recommended installation for the less experienced user.
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/index.adoc]:
```suggestion
The xref:{url_binary_installation}[Application installation] offers an easy and automated approach to set up Lisk Core. This includes almost completely automated update scripts, together with a selection of tools to assist with seamlessly maintaining a Lisk Core node. This is the recommended installation for the less experienced user.
```

File: modules/ROOT/pages/setup/source.adoc [hunk 15]
```diff
@@ -156,7 +156,7 @@ docker run --name lisk_core_db -p 5432:5432 -e POSTGRES_USER=lisk -e POSTGRES_PA
 ----
 
 This will install PostgreSQL version 10 (`postgres:10`) in a container with name `lisk_core_db` and binds the port `5432` of the container with the same port of the machine.
-As environment variables `POSTGRES_USER=lisk` is exposed in order to create the Lisk user, and `POSTGRES_PASSWORD=password` is used to set the password for the lisk user.
+As environment variables `POSTGRES_USER=lisk` is exposed in order to create the Lisk user, and `POSTGRES_PASSWORD=password` is used to set the password for the Lisk user.
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/source.adoc]:
```suggestion
As environment variables `POSTGRES_USER=lisk` is exposed in order to create the Lisk user, and `POSTGRES_PASSWORD=password` is used to set the password for the `lisk` user.
```

File: modules/ROOT/pages/setup/source.adoc [hunk 16]
```diff
@@ -219,8 +219,8 @@ sudo pg_createcluster --locale en_US.UTF-8 --start 10 main
 ----
 
 Create a new database user called `lisk` and grant it rights to create databases.
-Then create the database with the lisk user as owner.
-In the last step, define the password for the lisk user:
+Then create the database with the Lisk user as the owner.
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/source.adoc]:
```suggestion
Then create the database with the `lisk` user as the owner.
```

File: modules/ROOT/pages/setup/source.adoc [hunk 17]
```diff
@@ -219,8 +219,8 @@ sudo pg_createcluster --locale en_US.UTF-8 --start 10 main
...
+In the last step, define the password for the Lisk user:
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/source.adoc]:
```suggestion
In the last step, define the password for the `lisk` user:
```

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/management/logs.adoc (hunk 4 above)
Comment (chris529) [reply modules/ROOT/pages/management/logs.adoc]: Ah yes, of course. 👍

File: modules/ROOT/pages/interact-with-the-api.adoc (hunk 1 above)
Comment (chris529) [reply modules/ROOT/pages/interact-with-the-api.adoc]: It should be fixed now 👍

**Review by Tschakki — APPROVED** [review]
Thank you!

**Pull request merged**